src/ucp/core/ucp_request.h

```diff
@@ -64,12 +64,13 @@ enum {
  */
 enum {
     UCP_RECV_DESC_FLAG_FIRST    = UCS_BIT(0),
-    UCP_RECV_DESC_FLAG_LAST     = UCS_BIT(1),
-    UCP_RECV_DESC_FLAG_EAGER    = UCS_BIT(2),
-    UCP_RECV_DESC_FLAG_SYNC     = UCS_BIT(3),
-    UCP_RECV_DESC_FLAG_RNDV     = UCS_BIT(4),
-    UCP_RECV_DESC_FLAG_UCT_DESC = UCS_BIT(5),
-    UCP_RECV_DESC_FLAG_OFFLOAD  = UCS_BIT(6)
+    UCP_RECV_DESC_FLAG_MIDDLE   = UCS_BIT(1),
```

Comment:
pls don't change anything on RX side for now - i want to make a different change in the receive flags (will have flags for ONLY, EAGER, RNDV)  so it would just create conflicts..

Reply:
ok, rolled back (new flag is added to tail)
new flag is required to let know eager handler about new header type

Reply:
the level is wrong, debug should not be for every packet